aea/cli/init.py

```diff
@@ -38,11 +42,56 @@
 from aea.configurations.base import PublicId
 
 
-def _registry_init(author):
-    username = author
+def _local_init(author: Optional[str] = None) -> str:
```

Comment:
Maybe it's better to call it `_validate_author`?

Reply:
for symmetry with _registry_init, i think it makes sense to keep that name, although i agree it would be a candidate too